homeassistant/components/swiss_public_transport/coordinator.py

```diff
@@ -74,14 +77,21 @@ def nth_departure_time(self, i: int) -> datetime | None:
         return None
 
     async def _async_update_data(self) -> list[DataConnection]:
+        return await self.fetch_connections(limit=CONNECTIONS_COUNT)
+
+    async def fetch_connections(self, limit: int) -> list[DataConnection]:
+        """Fetch connections using the opendata api."""
+        self._opendata.limit = limit
         try:
             await self._opendata.async_get_data()
+        except OpendataTransportConnectionError as e:
+            _LOGGER.warning("Connection to transport.opendata.ch cannot be established")
```

Comment:
Coordinator already does logging so these should be removed (can be done in a separate PR)

Reply:
separate PR